pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -484,6 +491,53 @@ func (mp *MetaPatch) Validate() ([]metav1.StatusCause, bool) {
 	return causes, len(causes) == 0
 }
 
+// Counter is a Priority SortType
+const Counter string = "counter"
```

Comment:
```suggestion
const GameServerAllocationCounter string = "Counter"
```

Nit: Have all const values at the top of the file.

Since the package space is shared.

Also just realised a inconsistency in the design - values are always capitalised. "Packed", "Rolling", "Ready" etc. We should be consistent with the rest of Agones here too.

(Same for below)